doc/source/serve/advanced-guides/managing-java-deployments.md

```diff
@@ -3,6 +3,8 @@
 
 :::{warning}
 Java API support is an experimental feature and subject to change.
+
+Java API support on KubeRay is not currently prioritized.
```

Comment:
Not sure as an user if they care about it being prioritized or not. Maybe we can just say it's not currently supported. But fine either way :)

Reply:
+1

Reply:
If we're not planning to make improvements and test the API, I'd prefer to say `not supported`/`not prioritized` and leave it at that. Otherwise, we should add tests for the "supported" parts to make sure future changes don't affect them.